internal/workload/podman/podman.go

```diff
@@ -316,3 +323,56 @@ func (p *podman) GenerateSystemdService(podId string, monitoringInterval uint) (
 
 	return svc, nil
 }
+
+// Retrieve all pods logs and send that to the given io.Writer
+func (p *podman) Logs(podID string, res io.Writer) (context.CancelFunc, error) {
+
+	podInfo, err := pods.Inspect(p.podmanConnection, podID, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	opts := containers.LogOptions{
+		Follow: &boolTrue,
+		Stderr: &boolTrue,
+		Stdout: &boolTrue,
+		Tail:   &logTailLines,
+	}
+
+	ctx, cancel := context.WithCancel(p.podmanConnection)
+	stdoutCh := make(chan string, 100)
+	stderrCh := make(chan string, 100)
+
+	for _, container := range podInfo.Containers {
+		go func(name string) {
+			err := containers.Logs(ctx, name, &opts, stdoutCh, stderrCh)
+			if err != nil {
+				log.Errorf("cannot get container '%v' logs: %v", name, err)
+			}
+		}(container.ID)
+	}
+
+	go func() {
+		for {
+			select {
+			case line := <-stdoutCh:
```

Comment:
why not to use multi value case? Lines 359-362 and 364-267 are identical.

Reply:
select does not have multiple value cases no?

Reply:
right, only switches have multi value cases